### ISSUE TITLE
Unable to launch Chrome Browser of version 73 

### ISSUE THREAD

**Issue by purplekiz17** [bonigarcia/webdrivermanager#317]
Hi im unable to launch the chrome browser of ver 73 always getting the error as stated below : 

Chrome Browser version = Version 73.0.3683.75 (Official Build) (64-bit)
selenium-java = 3.141.59
webdrivermanager = 3.3.0

Note : Firefox works fine for me.
--------------------------------------------------------------------------------------------------------
Starting ChromeDriver 74.0.3729.6 (255758eccf3d244491b8a1317aa76e1ce10d57e9-refs/branch-heads/3729@{#29}) on port 15796
Only local connections are allowed.
Please protect ports used by ChromeDriver and related test frameworks to prevent access by malicious code.
Exception in thread "main" org.openqa.selenium.SessionNotCreatedException: session not created: This version of ChromeDriver only supports Chrome version 74
  (Driver info: chromedriver=74.0.3729.6 (255758eccf3d244491b8a1317aa76e1ce10d57e9-refs/branch-heads/3729@{#29}),platform=Windows NT 10.0.17134 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 3.86 seconds
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'INBAHRPC02609', ip: '10.53.34.67', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_181'
Driver info: driver.version: ChromeDriver
--------------------------------------------------------------------------------------------------------
![Maven Chrome Error](https://user-images.githubusercontent.com/18030019/54361714-a0958f80-468d-11e9-9258-7a1e56e3fb3b.jpg)

**Comment by jacsamell** [comment]
Updating to the latest version of webdrivermanager (3.3.0) fixed this for me. 
The older version wasn't aware of Chrome 73.

**Comment by purplekiz17** [comment]
@jacsamell it still shows a error for me and my webdrivermanager = 3.3.0

**Comment by bonigarcia** [comment]
See my comment in #318.

**Comment by medy75** [comment]
I have a same issue event when using WDM 3.3.0
I tried 
```
WebDriverManager.chromedriver().clearPreferences();
WebDriverManager.chromedriver().setup();
```
Still getting
```
org.openqa.selenium.SessionNotCreatedException: session not created: This version of ChromeDriver only supports Chrome version 74
  (Driver info: chromedriver=74.0.3729.6 (255758eccf3d244491b8a1317aa76e1ce10d57e9-refs/branch-heads/3729@{#29}),platform=Mac OS X 10.14.3 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 1.11 seconds
```

**Comment by purplekiz17** [comment]
from where can i download chrome browser ver = 74

**Comment by AphiweG** [comment]
Can someone please help to resolve this error: Exception in thread "main" java.lang.IllegalStateException: The driver executable is a directory: C:\Users\aphiw\Downloads\Drivers\Chromedriver

**Comment by robintyagi12** [comment]
There are 2 reasons when this issue may occur.

1. Case when WebDriverManager is not in use and somehow chromedriver.exe binary was updated which supports chrome browser version 74 but OS having browser as 72. In this case you would need to update binary compatible to browser version.

2. When WebDriverManager is in use which downloads browser binary automatically. Its tries to download latest chromedriver binary which may not be compatible to your chrome browser installed version let say 72 (my guess). If so, in that case we have to tell WebDriverManager to download chromedriver binary which would be compatible to your browser version 72. For doing so, please take a reference of below line and do update your code.

WebDriverManager.chromedriver().version("2.46").setup();

Note - We have added version("2.46") that means WebDriverManager would find out this specific version of chromedriver binary.

Let's know if this resolve your problem.

**Comment by AlyyMagdy** [comment]
> Can someone please help to resolve this error: Exception in thread "main" java.lang.IllegalStateException: The driver executable is a directory: C:\Users\aphiw\Downloads\Drivers\Chromedriver

You should include the full path including the name of the file.

To be like that :
C:\Users\aphiw\Downloads\Drivers\Chromedriver\chromedriver.exe

**Comment by bhavinshah09** [comment]
> There are 2 reasons when this issue may occur.
> 
> 1. Case when WebDriverManager is not in use and somehow chromedriver.exe binary was updated which supports chrome browser version 74 but OS having browser as 72. In this case you would need to update binary compatible to browser version.
> 2. When WebDriverManager is in use which downloads browser binary automatically. Its tries to download latest chromedriver binary which may not be compatible to your chrome browser installed version let say 72 (my guess). If so, in that case we have to tell WebDriverManager to download chromedriver binary which would be compatible to your browser version 72. For doing so, please take a reference of below line and do update your code.
> 
> WebDriverManager.chromedriver().version("2.46").setup();
> 
> Note - We have added version("2.46") that means WebDriverManager would find out this specific version of chromedriver binary.
> 
> Let's know if this resolve your problem.

I think my case is I so how do I fix it ? Please Help !!

**Comment by bonigarcia** [comment]
This should be fixed in version 3.6.0, recently released.

**Comment by Leigionrupak** [comment]
Does not work with 3.6.0 too?
![Screenshot (23)](https://user-images.githubusercontent.com/21272671/58449929-671cde00-812c-11e9-8b03-ebf0c469b55b.png)
